tox.ini

```diff
@@ -65,7 +65,7 @@ commands =
   coverage3 report
 
 [testenv:docs]
-basepython = python3
+basepython = python3.7
```

Comment:
I will undo this before we merge, I just need this to build locally, I'm hitting an error when trying to build with 3.10 that I don't feel like debugging right now.

Reply:
Done in: https://github.com/Qiskit/qiskit-terra/pull/7828/commits/0ba05920e7904b272032e39745feaa3b67d2cff4